ops/charm.py

```diff
@@ -525,7 +728,17 @@ def __init__(self, role: RelationRole, relation_name: str, raw: dict):
 
 
 class StorageMeta:
-    """Object containing metadata about a storage definition."""
+    """Object containing metadata about a storage definition.
+
+    Attributes:
+    storage_name: Name of storage
+    type: Storage type
+    description: A text description of the storage
+    read_only: Is the storage read only
```

Comment:
Possibly rephrase as "read_only: Whether or not the storage is read only"

I only suggest this because it seems off to have a question posed without a question mark.